pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -297,70 +297,88 @@ public static Schema getTableSchema(@Nonnull ZkHelixPropertyStore<ZNRecord> prop
    */
   public static List<OfflineSegmentZKMetadata> getOfflineSegmentZKMetadataListForTable(
       ZkHelixPropertyStore<ZNRecord> propertyStore, String tableName) {
```

Comment:
Should the signature include @Nonnull for propertyStore?

Reply:
There was a discussion before about how to use the **@Nullable** and **@NonNull** annotation. Because almost all arguments should be non-null, we decide to only use **@Nullable** annotation to mark arguments or return value that could be null.